.travis.yml

```diff
@@ -1,5 +0,0 @@
-language: python
-python:
-  - "2.7"
-install: "pip install -r requirements.txt"
-script: ./check_links.sh
```

Comment:
Would it not make sense to leave this in place to verify that all the redirect links are valid?